pkg/vcdcsiclient/disks.go

```diff
@@ -523,9 +522,9 @@ func (client *Client) DetachVolume(vm *govcd.VM, diskName string) error {
 	return nil
 }
 
-func (client *Client) GetRDEPersistentVolumes() ([]string, string, *swaggerClient.DefinedEntity, error) {
-	defEnt, _, etag, err := client.VCDClient.APIClient.DefinedEntityApi.GetDefinedEntity(context.TODO(),
-		client.VCDClient.ClusterID)
+func (diskManager *DiskManager) GetRDEPersistentVolumes() ([]string, string, *swaggerClient.DefinedEntity, error) {
```

Comment:
Can we move the RDE functions to an `rde.go` file? I am half thinking that this could also be moved to the common core, but not 100% sold on it. What do you think @sahithi @Anirudh9794